StardewCropCalc/com.elizabethburgessamt.stardewcropcalc/seeds/Seed.java

```diff
@@ -2,4 +2,22 @@
 
 public class Seed {
 	//TODO: Create proper data object
+	private string crop;
+	private int growth;
+	private int regrowth;
+	private int cost;
+	private int sell;
+	private boolean trellis;
+	
+	public Seed(string crop, int growth, int regrowth, int cost, int sell, string trellis){
+		this.crop = crop;
+		this.growth = growth;
+		this.regrowth = regrowth;
+		this.cost = cost;
+		this.sell = sell;
+		if (trellis == "TRUE"){
+			this.trellis = true;
+		}
+		else { this.trellis = false; }

```

Comment:
Try to follow the [Google Java Style Guide](http://google.github.io/styleguide/javaguide.html).  This means else on the same line as }, no inline code statements generally, keep in mind spacing and everything with parentheses and brackets ) {, for example.